server/index.js

```diff
@@ -22,17 +22,17 @@ app.get('/app/:gameid', (req, res) => {
 
 router.get('/gamereviews/:gameid', async (req, res) => {
   let { gameid } = req.params;
-  if (parseInt(gameid) <= 0 || parseInt(gameid) > 100) {
+  console.log(req.originalUrl)
+  if (parseInt(gameid) <= 0 || parseInt(gameid) > 30000) {
     res.status(400).json({ error: 'Invalid game ID. Please use a number between 1 and 100.' });
```

Comment:
Either way, I think this error message needs updating.

Reply:
I will definitely fix this, nice catch